jekyll/_cci2/env-vars-set.adoc

```diff
@@ -0,0 +1,248 @@
+---
+version:
+- Cloud
+- Server 3.x
+- Server 4.x
+---
+= Set an Environment Variable
+:page-layout: classic-docs
+:page-liquid:
+:page-description: How to set environment variables for use in your CircleCI pipelines. 
+:icons: font
+:toc: macro
+:toc-title:
+
+There are several ways to use environment variables in CircleCI to provide variety in scope and authorization level.
+
+[#set-an-environment-variable-in-a-shell-command]
+== Set an environment variable in a shell command
+
+While CircleCI does not support interpolation when setting environment variables, it is possible to set variables for the current shell by <<env-vars#using-parameters-and-bash-environment,using `BASH_ENV`>>. This is useful for both modifying your `PATH` and setting environment variables that reference other variables.
+
+```yaml
+version: 2.1
+
+jobs:
+  build:
+    docker:
+      - image: smaant/lein-flyway:2.7.1-4.0.3
+        auth:
+          username: mydockerhub-user
+          password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
+    steps:
+      - run:
+          name: Update PATH and Define Environment Variable at Runtime
+          # Add source command to execute code and make variables 
+          # available in current step.
+          command: |
+            echo 'export PATH=/path/to/foo/bin:"$PATH"' >> "$BASH_ENV"
+            echo 'export VERY_IMPORTANT=$(cat important_value)' >> "$BASH_ENV"
+            source "$BASH_ENV"
+```
+
+**Note**:
```

Comment:
There are a couple of notes formatted like this, could you switch them to use the `NOTE:` admonition style?